src/main/java/io/jenkins/plugins/analysis/core/columns/IssuesTotalColumn.java

```diff
@@ -264,12 +264,18 @@ public ListBoxModel doFillTypeItems() {
             this.url = url;
         }
 
-        AnalysisResultDescription(final ResultAction result, final LabelProviderFactory labelProviderFactory) {
+        AnalysisResultDescription(final ResultAction result, final LabelProviderFactory labelProviderFactory,
+                final StatisticProperties type) {
             StaticAnalysisLabelProvider labelProvider = labelProviderFactory.create(result.getId(), result.getName());
             name = labelProvider.getLinkName();
             icon = labelProvider.getSmallIconUrl();
             total = result.getResult().getTotalSize();
-            url = result.getUrlName();
+
+            String typeUrl = type.name().split("_")[0];
+            if (!typeUrl.matches("NEW|FIXED")) {
+                typeUrl = "";
+            }
```

Comment:
Make this more object oriented ([tell don't ask](https://martinfowler.com/bliki/TellDontAsk.html)). 

Add a `getUrl` method in StatisticsProperties. And there add an additional constructor parameter that provides the URL.  E.g. in https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/main/java/io/jenkins/plugins/analysis/core/util/IssuesStatistics.java#L243: 
```
NEW_HIGH(Messages._Statistics_New_High(), IssuesStatistics::getNewHighSize, "new/high"),
```
There is no need to compute it using an algorithm.

Reply:
Maybe it is even better to provide a method `getUrl` that has an extra parameter prefix that handles the problem that the URL should not have a slash as last element. Then we can invoke it with
```suggestion
url = type.getUrl(result.getOwner().getNumber() + "/" + result.getUrlName())
```